control-plane-operator/controllers/hostedcontrolplane/ignition/reconcile.go

```diff
@@ -89,20 +96,20 @@ func SetMachineConfigLabels(mc *mcfgv1.MachineConfig) {
 	}
 }
 
-func reconcileImageContentPolicyIgnitionConfigMap(cm *corev1.ConfigMap, imageDigestMirrorSet *configv1.ImageDigestMirrorSet, ownerRef config.OwnerRef) error {
+func reconcileImageContentTypeIgnitionConfigMap(cm *corev1.ConfigMap, imageContentType client.Object, ownerRef config.OwnerRef) error {
 	scheme := runtime.NewScheme()
-	err := configv1.Install(scheme)
+	err := v1alpha1.Install(scheme)
```

Comment:
you should install both v1alpha1 and configv1 (also, just noticed... v1alpha1 is probably not a good package alias. You should use something like operatorv1alpha1)